pkg/codegen/main.go

```diff
@@ -19,6 +19,7 @@ import (
 
 func main() {
 	controllergen.Run(args.Options{
+		ImportPackage: "github.com/rancher/wrangler/v2/pkg/generated",
 		OutputPackage: "github.com/rancher/wrangler/pkg/generated",
```

Comment:
Does the `OutputPackage` also need the `v2` added?

Reply:
`OutputPackage` determines where in the repo to put the generated files. I had the same question before and tested it out locally, and when you change the output to `v2` it creates a new local v2 directory for the generated file. I think it would be more clear if the variable name was `OutputPath` instead of package.

Reply:
That makes sense.